backend/routes/group.py

```
@@ -48,3 +53,81 @@ def get_university_info(university_name: str, db: Session = Depends(get_db)):
+    for uni in universities:
+        dept_preview = uni.departments[:limit_departments]
+        result.append({
+            "id": uni.id,
+            "name": uni.name,
+            "departments": dept_preview,
+            "has_more_departments": len(uni.departments) > limit_departments
+        })
+
+    return result
+
+@router.get("/{university_id}/departments", response_model=List[str])
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)